site.py

```diff
@@ -0,0 +1,233 @@
+from pathlib import Path
+import os
+import sys
+import argparse
+import subprocess
+from typing import Union, List, Optional, Mapping
+
+IMAGE_NAME="airflow-site"
+CONTAINER_NAME="airflow-site-c"
+SITE_DIST="landing-pages/dist"
+THEME_GEN="sphinx_airflow_theme/sphinx_airflow_theme/static/_gen"
+RunCommandResult = Union[subprocess.CompletedProcess, subprocess.CalledProcessError]
+
+
+def run_command(cmd: List[str],
+    env: Optional[Mapping[str, str]] = None,
+    cwd: Optional[Path] = None,
+    input: Optional[str] = None,
+    check: bool = True,
+     **kwargs,
+)-> RunCommandResult:
+    workdir: str = str(cwd) if cwd else os.getcwd()
+    try:
+        cmd_env = os.environ.copy()
+        cmd_env.setdefault("HOME", str(Path.home()))
+        if env:
+            cmd_env.update(env)
+        return subprocess.run(cmd, input=input, check=check, env=cmd_env, cwd=workdir, **kwargs)
+    except subprocess.CalledProcessError as ex:
+        if ex.stdout:
+            print(ex.stdout)
+        if ex.stderr:
+            print(ex.stderr)
+        return ex
```

Comment:
Do we need to return an error here? Would it be better to raise it?

Reply:
This is a common method to execute the script via Python subprocess. If I could return the error, then I can log the error message according to the command executed. If you have any other options please let me know @turbaszek